chart/templates/configmap.yaml

```diff
@@ -23,6 +23,7 @@ data:
         slack:
           enabled: {{ .Values.controller.notifiers.slack.enabled }}
           tokenFilepath: {{ .Values.controller.notifiers.slack.tokenFilepath | quote }}
+          mirrorSlackChannelId: {{ .Values.controller.notifiers.slack.mirrorSlackChannelId | quote }}
```

Comment:
What do you think of this taking a list of channels, rather than single? Or naming it `notification_channels`?

Reply:
I was more using this as a temporary parameter to broadcast only to our team and remove it when the feature will be deployed, however if you think this would be useful in a more permanent way, yes.